### DISCUSSION TITLE
Config options related to saving logs and screen to file

### DISCUSSION THREAD

**Discussion by trufae** [radareorg/radare2#21143] [Q&A]
Those two were marked to be changed for 5.8.0, but I get no feedback for that change and i'm not 100% sure about it. Please share your thoughs in this thread.

<img width="676" alt="Screenshot 2022-12-08 at 13 07 00" src="https://user-images.githubusercontent.com/6431515/206442887-e2f97ec0-52b5-48e4-8d4c-fa6b4dc35854.png">

**Comment by alex-bender** [comment]
what would be in log?
is "file.output" connected with "scr.tee"? it's confusing from this help message
